akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -810,6 +810,58 @@ final class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends
     new Flow(delegate.conflate(seed.apply)(aggregate.apply))
 
   /**
+   * Allows a faster upstream to progress independently of a slower subscriber by aggregating elements into chunks
+   * until the subscriber is ready to accept them. For example an aggregate step might store received elements in
+   * an array up to the allowed max limit if the upstream publisher is faster.
+   *
+   * This element only rolls up elements if the upstream is faster, but if the downstream is faster it will not
+   * duplicate elements.
+   *
+   * '''Emits when''' downstream stops backpressuring and there is an aggregated element available
+   *
+   * '''Backpressures when''' there are `max` aggregated elements and 1 pending element and downstream backpressures
+   *
+   * '''Completes when''' upstream completes and there is no aggregated/pending element waiting
+   *
+   * '''Cancels when''' downstream cancels
+   *
```

Comment:
maybe add a reference to conflate, which is similar, but without an upper limit. Alternative solution is to express conflate in terms of this one, making the cost function constant zero, and making the max Long.MaxValue. Another possible improvement is to make `seed` default to the identity function. @rkuhn you work on something similar with conflate, do you have any other opinion?